testing/data/W29.py

```diff
@@ -14,8 +14,6 @@ class Foo(object):
 #: W191 W292 noeol
 if False:
 	pass  # indented with tabs
-#: W292:1:36 noeol
-# This line doesn't have a linefeed
```

Comment:
there was a bug here!  in python 3.8 this was reported 3 times.  I didn't bother fixing the bug since it'll be "magically fixed" in python3.9+

I moved the test to its own file